src/components/ebay-tab/migrator.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function migrator(el, context) {
+    context.deprecate('<ebay-tab> has been renamed to <ebay-tabs>');
+    const oldAttribute = 'on-tab-select';
+    const newAttribute = 'on-tabs-select';
+    el.setTagName('ebay-tabs');
+    if (el.hasAttribute(oldAttribute)) {
+        const attribute = el.getAttribute(oldAttribute);
+        el.removeAttribute(oldAttribute);
+        attribute.name = newAttribute;
```

Comment:
You shouldn't need to remove then add the attribute, mutating the `.name` should suffice.

Reply:
Done